lib/vagrant-service-manager/command.rb

```diff
@@ -101,36 +110,49 @@ def print_help(config = {})
         exit config[:exit_status]
       end
 
-      def check_if_a_service_is_running?(service)
+      def service_running?(service)
         command = "systemctl status #{service}"
         with_target_vms(nil, {:single_target=>true}) do |machine|
           return machine.communicate.test(command)
         end
       end
 
-      def print_all_provider_info
-        @env.ui.info I18n.t('servicemanager.commands.env.nil')
+      def service_status(service)
+        return I18n.t('servicemanager.commands.status.status.running') if service_running?(service)
```

Comment:
`kubernetes` service is a special case, where user enters `kuberenetes` on STDIN however in the box there is no such service as `kubernetes`. It has to be taken care explicitly as done in https://github.com/projectatomic/vagrant-service-manager/pull/176 . Although I can update PR #76 after this PR is merged, but I mentioned since this piece of code is updated in another PR.